### PR TITLE
Fix the issue of configuration modification failure

### LINKED ISSUE #1543: Fix Configuration Management function test problems
### What happened?

The configuration management function test is faulty

### How can we reproduce it (as minimally and precisely as possible)?

The problem can be reproduced when the latest backend code is used to add, modify, and query configurations.

### Anything else we need to know?

nothing

### Sermant version

<details>
1.4.1
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```
</details>

### PR DESCRIPTION
**Pull request by hanbingleixue**
**What type of PR is this?**

Bug

**What this PR does / why we need it?**

1、Fix the issue that configurations fail to be modified in the Kie environment of the configuration center.
2、Fix the problem that the old version of Nacos gets the wrong service status
3、Fix the problem that Zookeeper fails to query the configuration of other plug-ins.

**Which issue(s) this PR fixes？**

Fixes #1543 

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/sermant-io/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/dynamicconfig/zookeeper/ZooKeeperClient.java [hunk 1]
```diff
@@ -171,7 +171,12 @@ private Map<String, List<String>> fuzzyGetConfigListByGroupAndKey(String key, St
             if (!child.contains(nodeName)) {
                 continue;
             }
-            String childPath = parentNodePath + child;
+            String childPath;
+            if (parentNodePath.endsWith(String.valueOf(ZK_PATH_SEPARATOR))) {
+                childPath = parentNodePath + child;
+            } else {
+                childPath = parentNodePath + ZK_PATH_SEPARATOR + child;
+            }
```
Comment (lilai23) [inline sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/dynamicconfig/zookeeper/ZooKeeperClient.java:179]: just childPath = parentNodePath + ZK_PATH_SEPARATOR + child  maybe ok

Comment (hanbingleixue) [reply sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/dynamicconfig/zookeeper/ZooKeeperClient.java:179]: parentNodePath may be /, can't just do childPath = parentNodePath + ZK_PATH_SEPARATOR + child

**Review by lilai23 — APPROVED** [review]

**Review by lilai23 — DISMISSED** [review]

File: sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/dynamicconfig/kie/client/kie/KieClient.java [hunk 2]
```diff
@@ -153,7 +153,14 @@ public boolean publishConfig(String key, Map<String, String> labels, String cont
 
     @Override
     public boolean publishConfig(String key, String group, String content) {
-        return this.publishConfig(key, LabelGroupUtils.resolveGroupLabels(group), content, true);
+        final Optional<String> keyIdOptional = this.getKeyId(key, group);
+        if (!keyIdOptional.isPresent()) {
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/dynamicconfig/kie/client/kie/KieClient.java]: if(keyIdOptional.isPresent()) {
    return ...;
}
return ....;

Comment (hanbingleixue) [reply sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/dynamicconfig/kie/client/kie/KieClient.java]: done

**Review by daizhenyu — APPROVED** [review]

**Review by lilai23 — APPROVED** [review]

**Pull request merged**